src/jit/gentree.cpp

```diff
@@ -5954,7 +5954,8 @@ GenTreePtr          Compiler::gtCloneExpr(GenTree * tree,
 
 #ifdef FEATURE_READYTORUN_COMPILER
             copy->gtFptrVal.gtEntryPoint = tree->gtFptrVal.gtEntryPoint;
-            copy->gtFptrVal.gtDelegateCtor = tree->gtFptrVal.gtDelegateCtor;
+            copy->gtFptrVal.gtLdftnResolvedToken = new (this, CMK_Unknown) CORINFO_RESOLVED_TOKEN;
+            memcpy(copy->gtFptrVal.gtLdftnResolvedToken, tree->gtFptrVal.gtLdftnResolvedToken, sizeof(CORINFO_RESOLVED_TOKEN));
```

Comment:
Why is gtLdftnResolveToken a pointer to a CORINFO_RESOLVED_TOKEN instead of an object directly? Then you wouldn't need to do a 'new'. In any case, you don't need the memcpy; even in this case it seems you could do "*copy->gtFptrVal.gtLdftnResolvedToken = *tree->gtFptrVal.gtLdftnResolvedToken"